en/lessons/advanced/escripts.md

```diff
@@ -11,11 +11,24 @@ To build executables in Elixir we will be using escript. Escript produces an exe
 
 ## Getting Started
 
-To create an executable with escript there are only a few things we need to do: implement a `main/1` function and update our Mixfile.
+For a given Mix project, to create an executable with escript there are only a few things we need to do: implement a `main/1` function and update our Mixfile.
+
+We'll begin by generating a project with a supervision tree:
+
+```shell
+$ mix new example_app
+$ cd example_app
+```
+
+We'll start by creating a module to serve as the entry point to our executable.  This is where we'll implement `main/1`. This module can be a separate file within the `lib` directory, or simply added to the default module like shown below.
 
-We'll start by creating a module to serve as the entry point to our executable.  This is where we'll implement `main/1`:
 
 ```elixir
+# ./lib/example_app.ex
+defmodule ExampleApp do
```

Comment:
Please remove this.  We do not include this sort of placeholder in any other examples.